### DISCUSSION TITLE
Release v2.12.1

### DISCUSSION THREAD

**Discussion by mika-f** [mika-f/blender-drag-and-drop#73] [Announcements]
## What's Changed
* Fix: When the file extension is *.FBX, the file cannot be imported (#71)
* Release v2.12.1 by @mika-f in https://github.com/mika-f/blender-drag-and-drop/pull/72

**Full Changelog**: https://github.com/mika-f/blender-drag-and-drop/compare/v2.12.0...v2.12.1

## NOTES

* Restart Blender after updating add-on. Due to limitations of Windows, changes will not take effect until restart.
* **This release will be the final release of the v2.x series. Blender 4.0.1 and later is only supported in v3.x.**

<hr /><em>This discussion was created from the release <a href='https://github.com/mika-f/blender-drag-and-drop/releases/tag/v2.12.1'>Release v2.12.1</a>.</em>

**Comment by crizzpy** [comment]
thanks for updating, love your addon!
